test/selenium/signInPage.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var SignedInPageView = require('./lib/view/signed_in/view.js');
+var signedInPageView;
+
+function SignInPage(driver) {
```

Comment:
After reading your next comment, that would make sense, as it was as the beginning, so that it will be in charge of initializing each view.
Will change the name